### PR TITLE
#3221 update category text style

### PR DESCRIPTION
**Pull request by nabilIbm**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/common-canvas/src/palette/palette.scss [hunk 1]
```diff
@@ -230,13 +230,12 @@ $palette-border-color: $ui-03;
 }
 
 .palette-flyout-category-text {
-	line-height: 46px;  /* Must be same as containing div height */
-	overflow: hidden;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	display: flex;
```
Comment (tomlyn) [inline canvas_modules/common-canvas/src/palette/palette.scss:233]: Why dos display need to be flex? If I take it out it seems to display OK. Please don't add fields into the CSS unless they are necessary.

File: canvas_modules/common-canvas/src/palette/palette.scss [hunk 2]
```diff
@@ -230,13 +230,12 @@ $palette-border-color: $ui-03;
...
+	width: 100%;
+	overflow-wrap: break-word;
+	line-height: 1.3;
 	margin-left: 16px;
-	width: 135px; // We need to allow enough space for search numbers to appear eg "(12)"
-}
+ }
```
Comment (tomlyn) [inline canvas_modules/common-canvas/src/palette/palette.scss:238]: This closing bracket should not be indented?

**Review by tomlyn — CHANGES_REQUESTED** [review]
While making the requested changes can you make a new PR for this where the first commit in the PR has a message which follows the correct naming convention please? That is, using the Elyra Canvas issue number at the beginning of the message. BTW - It is the commit for the first message that is used in the log so even if you correct it for later commit messages they won't be included in the log when the commits are  merged together.

File: canvas_modules/common-canvas/src/palette/palette.scss (hunk 1 above)
Comment (nabilIbm) [reply canvas_modules/common-canvas/src/palette/palette.scss:233]: you are right !! I was trying flex-wrap in the first place and get elements gets wrapped  however it didn't work..white-space: nowrap did the job. so I had to remove flex.. hmmm let check what is the issue with caret icon

File: canvas_modules/common-canvas/src/palette/palette.scss [hunk 3]
```diff
@@ -230,12 +234,11 @@ $palette-border-color: $ui-03;
...
-	overflow: hidden;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	display: flex;
+	width: 100%;
+	overflow-wrap: break-word;
+	line-height: 1.3;
 	margin-left: 16px;
-	width: 135px; // We need to allow enough space for search numbers to appear eg "(12)"
 }
```
Comment (tomlyn) [inline canvas_modules/common-canvas/src/palette/palette.scss:242]: It still doesn't look the same. 

image

Also you have changed the hover behavior which need  to stay the same. The hover behavior now is strange ... 

PalDrag

**Comment by tomlyn** [comment]
These changes have been implemented as part of a different PR (which uses the correct naming convention for the commit message)

**Pull request closed without merging**